### PR TITLE
Tandem Search Pills

### PR DESCRIPTION
**Pull request by mjoyce91**
~~Relies on #884 (merge first)~~

- Conditional coloring for tandem search pills, to differentiate among Tandem 1, Tandem 2, and common
- Hover text for pills, so that the user knows which pills associate with which tandem user
- Different ordering of the pills based on individual vs tandem search
- Hide any `isToggle` filters from the Pill List
- Refactor some functions

### CODE REVIEW

File: src/Components/PillList/PillList.jsx [hunk 1]
```diff
@@ -1,25 +1,39 @@
...
-            key={`${item.codeRef}-${item.description}`}
+const PillList = ({ items, onPillClick }, { isTandemSearch }) => {
+  const ordering = ['description', 'code'];
+  if (isTandemSearch) ordering.splice(0, 0, 'isCommon', 'isTandem');
+  return (
+    <div className="pill-list-container">
+      {
+        // order items in their correct context
+        (orderBy(items.slice(), ordering))
+          // do not display toggles as pills
```
Comment (elizabeth-jimenez) [inline src/Components/PillList/PillList.jsx:15]: 🥇

File: src/Containers/Results/Results.jsx [hunk 2]
```diff
@@ -47,6 +47,14 @@ class Results extends Component {
     this.debounced = debounce(() => {});
   }
 
+  getChildContext() {
```
Comment (elizabeth-jimenez) [inline src/Containers/Results/Results.jsx:50]: 😎

**Comment by elizabeth-jimenez** [comment]
@mjoyce91 It looks really good and functions really well! The only small thing I have, and it may have been intentional, is the ordering of the pills. Did we intentionally want Tandem2's pills to come before Tandem1's pills?

File: src/Components/PillList/PillList.jsx [hunk 3]
```diff
@@ -1,25 +1,39 @@
...
-const PillList = ({ items, onPillClick }) => (
-  <div className="pill-list-container">
-    {
-      (items.slice().sort(pillSort))
-        .map(item =>
-          (<Pill
-            key={`${item.codeRef}-${item.description}`}
+const PillList = ({ items, onPillClick }, { isTandemSearch }) => {
+  const ordering = ['description', 'code'];
+  if (isTandemSearch) ordering.splice(0, 0, 'isCommon', 'isTandem');
```
Comment (elizabeth-jimenez) [inline src/Components/PillList/PillList.jsx:9]: 💭 ordering 🤔 :)

**Comment by elizabeth-jimenez** [comment]
- [ ] Conditional coloring for tandem search pills, to differentiate among Tandem 1, Tandem 2, and common

- [ ] Hover text for pills, so that the user knows which pills associate with which tandem user

- [ ] Different ordering of the pills based on individual vs tandem search

- [ ] Hide any isToggle filters from the Pill List

- [ ] Refactor some functions

**Comment by mjoyce91** [comment]
>  Different ordering of the pills based on individual vs tandem search

@elizabeth-jimenez is this not working for you?

**Comment by elizabeth-jimenez** [comment]
> > Different ordering of the pills based on individual vs tandem search
> 
> @elizabeth-jimenez is this not working for you?

Ah, it is, no worries! I wish there was a half check option. I just didn't check it off cause I wondering about the ordering https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/885#issuecomment-625892903

**Comment by mjoyce91** [comment]
> Did we intentionally want Tandem2's pills to come before Tandem1's pills?

@elizabeth-jimenez - good point, meant to fix that. Updating now

File: src/Components/PillList/PillList.jsx [hunk 4]
```diff
@@ -6,12 +6,16 @@ import Pill from '../Pill/Pill';
 
 const PillList = ({ items, onPillClick }, { isTandemSearch }) => {
   const ordering = ['description', 'code'];
-  if (isTandemSearch) ordering.splice(0, 0, 'isCommon', 'isTandem');
+  const orders = [];
+  if (isTandemSearch) {
+    ordering.splice(0, 0, 'isCommon', 'isTandem');
+    orders.splice(0, 0, 'asc', 'desc');
```
Comment (elizabeth-jimenez) [inline src/Components/PillList/PillList.jsx:12]: 😍

**Pull request merged**